src/command_line/indexing/bulk_index.rs

```diff
@@ -38,7 +40,7 @@ pub fn action(matches: &ArgMatches) -> eyre::Result<()> {
     let index_manager = IndexManager::new(storage_dir.deref(), false)?;
 
     let index = index_manager.open(index_name.deref())?;
-    let mut writer = index.writer(8 * 1024 * 1024)?;
```

Comment:
What drove this change?

Reply:
Some error about memory bytes needing to be greater than 15,000,000. I feel like we got a similar error in the past which prompted us to change it to `8 * 1024 * 1024`, so maybe just another tantivy update?